cmd/create/oidcconfig/cmd.go

```diff
@@ -112,21 +113,58 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
+	oidcConfigInput := buildOidcConfigInput(r)
 	oidcConfigStrategy, err := getOidcConfigStrategy(mode)
 	if err != nil {
 		r.Reporter.Errorf("%s", err)
 		os.Exit(1)
 	}
-	bucketUrl, privateKeyFilename := oidcConfigStrategy.execute(r)
+	oidcConfigStrategy.execute(r, oidcConfigInput)
 	if r.Reporter.IsTerminal() {
 		r.Reporter.Infof("To create a cluster with this oidc config, please run:\n"+
 			"rosa create cluster --sts --oidc-endpoint-url %s "+
-			"--bound-service-account-signing-key-path ./%s", bucketUrl, privateKeyFilename)
+			"--bound-service-account-signing-key-secret-name %s",
+			oidcConfigInput.BucketUrl, oidcConfigInput.BucketName)
+	}
+}
+
+type OidcConfigInput struct {
+	BucketName         string
+	BucketUrl          string
+	PrivateKey         []byte
+	PrivateKeyFilename string
```

Comment:
Why do we need a file name if we're using the secrets manager? I think you should generate the secret and keep it in memory until you send it to AWSSM. No need to store in a file.